README.md

```diff
@@ -29,12 +29,29 @@ resources:
     private_key: |
             ...
 
+- name: sync-resource-multiple
+  type: rsync-resource
+  source:
+    servers:
+      - server1
+      - server2
+    base_dir: /sync_directory
+    user : user
+    disable_md5_path: false
```

Comment:
This would be better named as disable version path (user doesn't necessarily need to know its MD5)